### PR TITLE
Add the test signing certificates for secure boot

### PR DESCRIPTION
**Pull request by xumia**
**- Why I did it**
Check in the test certificates to sign aboot image for secure boot, if the certificates not specified, the test certificates will be used. The certificates are only for test, not for product. 

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: build_image.sh [hunk 1]
```diff
@@ -150,7 +150,15 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     if [ "$SONIC_ENABLE_IMAGE_SIGNATURE" = "y" ]; then
         TARGET_CA_CERT="$TARGET_PATH/ca.cert"
         rm -f "$TARGET_CA_CERT"
-        [ -f "$CA_CERT" ] && cp "$CA_CERT" "$TARGET_CA_CERT"
+
+        # If the ca certificate does not exist, the test certificate will be used to sign the image
+        if [ ! -f "$CA_CERT" ]; then
+            TEST_CERT_PATH=files/image_config/secureboot/test-certs
```
Comment (qiluo-msft) [inline build_image.sh:156]: add `local` ?

File: files/image_config/secureboot/test-certs/ca.key [hunk 2]
```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (qiluo-msft) [inline files/image_config/secureboot/test-certs/ca.key:1]: Do you need this file in build process?

File: build_image.sh (hunk 1 above)
Comment (xumia) [reply build_image.sh:156]: Seems it cannot be changed to local, it only can be used in a function.

File: files/image_config/secureboot/test-certs/ca.key (hunk 2 above)
Comment (xumia) [reply files/image_config/secureboot/test-certs/ca.key]: Removed.

**Comment by qiluo-msft** [comment]
Retest vsimage please

**Comment by qiluo-msft** [comment]
Retest vsimage please

**Pull request merged**